CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to @dhl-uu/timing-util
+
+## Required software
+
+- Node.js
+- npm
+- git
+- [editorconfig plugin][editorconfig] for your favorite editor
```

Comment:
This seems to include an implicit expectation that contributors use an autoformatter or linter. If you have formatting expectations for contributors, it doesn't hurt to explicate those.